doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1235,6 +1235,10 @@ Other libraries
 
   * Added the :c:func:`date_time_now_local` function to the API.
 
+* :ref:`dk_buttons_and_leds_readme` library:
```

Comment:
Can we have the changelog entry into a separate commit? (I know we usually commit such small changes in one commit. However, in this context, we have a chance for a clear revert of this feature once the PDK v0.2.x support is dropped).